tools/__tasks__/compile/css/sass.js

```diff
@@ -1,45 +1,51 @@
 const compile = require('../../../compile-css');
 
+/** @type {import('listr2').ListrTask} */
 const task = {
-	description: 'Compile Sass',
-	task: [
-		{
-			description: 'Old IE',
-			task: () =>
-				compile('old-ie.*.scss', {
-					browsers: 'Explorer 8',
-					remify: false,
-				}),
-		},
-		{
-			description: 'IE9',
-			task: () =>
-				compile('ie9.*.scss', {
-					browsers: 'Explorer 9',
-				}),
-		},
-		{
-			description: 'Email',
-			task: () =>
-				compile('head.email-{article,front}.scss', {
-					remify: false,
-				}),
-		},
-		{
-			description: 'Modern',
-			task: () =>
-				compile('!(_|ie9|old-ie|*email-article|*email-front)*.scss'),
-		},
-		{
-			description: 'Inline',
-			task: () => compile('inline/*.scss'),
-		},
-		{
-			description: 'Atoms',
-			task: () => compile('atoms/*.scss'),
-		},
-	],
-	concurrent: true,
+	title: 'Compile Sass',
+	task: (ctx, task) =>
+		task.newListr(
+			[
+				{
+					title: 'Old IE',
+					task: () =>
+						compile('old-ie.*.scss', {
+							browsers: 'Explorer 8',
+							remify: false,
+						}),
+				},
+				{
+					title: 'IE9',
+					task: () =>
+						compile('ie9.*.scss', {
+							browsers: 'Explorer 9',
+						}),
+				},
+				{
+					title: 'Email',
+					task: () =>
+						compile('head.email-{article,front}.scss', {
+							remify: false,
+						}),
+				},
+				{
+					title: 'Modern',
+					task: () =>
+						compile(
+							'!(_|ie9|old-ie|*email-article|*email-front)*.scss',
+						),
+				},
+				{
+					title: 'Inline',
+					task: () => compile('inline/*.scss'),
+				},
+				{
+					title: 'Atoms',
+					task: () => compile('atoms/*.scss'),
+				},
+			],
+			{ concurrent: !!ctx.verbose ? false : true },
```

Comment:
Whats the reason to switch concurrency here?

Reply:
Previously, when `concurrent` was `true`, [`listrify` would read this value and convert it to `true` or `false` based on a series of check](https://github.com/guardian/frontend/blob/2b874f4c2b55033e1dc6f2b3144b120a965b27e1/tools/task-runner/runner.mjs#L131-L135). This makes it more explicit at the point of creation.